debian/Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM debian:stretch
+MAINTAINER Shaun Jackman <sjackman@gmail.com>
```

Comment:
I've forked from the ubuntu version, but it's indeed deprecated, and as it's a just a fork I prefer to let the name of the original author.